android/README.md

```diff
@@ -0,0 +1,177 @@
+# Introduction 
+
+This sample project covers:
+
+1. Setting up ImageKit iOS SDK
+2. Rendering images
+3. Setting authentication context for the SDK
+4. Applying common image manipulations
+5. Adding overlays to images
+6. Client-side file uploading
+
+# How to run locally
+
+This project was bootstrapped with [Android Studio](https://developer.android.com/studio/install) using [Kotlin](https://kotlinlang.org/). Head over to the link for installation if not already done.
+
```

Comment:
Should we mention the Gradle Version and Java Version to run the application as well?

Reply:
Doesn't seem necessary, since Android Studio itself gives the necessary suggestions to upgrade Gradle and JVM toolchains in case the build fails with the current versions.